### PR TITLE
feat: Rename custom performance metrics to custom peformance measurem…

### PR DESCRIPTION
**Pull request by matejminar**
Closes https://github.com/getsentry/sentry/issues/71226

Just renaming things now. We are waiting for result of this [DACI](https://www.notion.so/sentry/DACI-Remove-generic-setMeasurement-in-favor-of-Metrics-de6c8ac7b37d4c288c2fc6c135699106) and add deprecation notes then if needed.

### CODE REVIEW

File: includes/custom-measurements-units-disclaimer.mdx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<Note>
+
+Currently, unit conversion is only supported once the data has already been stored. This means that, for example, `('myMetric', 60, 'second')` and `('myMetric', 3, 'minute')` would not be aggregated, but rather stored as two separate measurements. To avoid this, make sure to use a consistent unit when recording a custom measurement.
```
Comment (shellmayr) [inline includes/custom-measurements-units-disclaimer.mdx:3]: Should the names rather be myMeasurement corresponding with the rename?

**Review by shellmayr — APPROVED** [review]

**Review by obostjancic — APPROVED** [review]
lgtm

File: includes/custom-measurements-units-disclaimer.mdx (hunk 1 above)
Comment (matejminar) [reply includes/custom-measurements-units-disclaimer.mdx]: good catch!

**Review by gggritso — APPROVED** [review]

**Review by ale-cota — APPROVED** [review]

File: includes/custom-measurements-disclaimer.mdx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<Alert level="warning">
+
+For a more flexible solution where your measurements are not tied to transactions and not limited to 10 per transaction, we recommend you use [custom metrics](https://docs.sentry.io/product/metrics/) instead.
```
Comment (vivianyentran) [inline includes/custom-measurements-disclaimer.mdx:3]:
```suggestion
Custom performance measurements are limited to 10 per transaction. If you have more than 10 measurements per transaction, the list will be truncated. For a more flexible solution, we recommend using [custom metrics](https://docs.sentry.io/product/metrics/) instead.
```

File: docs/product/performance/metrics.mdx [hunk 3]
```diff
@@ -99,28 +99,25 @@ For each project, you can configure how [Apdex](#apdex) and [User Misery](#user-
...
 - Create visualizations in **Dashboards**
 - Query and add columns for them in **Discover**
-- View a transaction event detail (see image below)
-
-![Custom metrics in an event detail](./img/custom-metrics-performance.png)
-
-> _Custom metrics data displayed in the sidebar of a transaction event detail under the "Custom Performance Metrics" heading._
 
-You can define up to 10 custom performance metrics per transaction. If more custom performance metrics are included in a transaction event, the list will be truncated.
+You can define up to 10 custom performance measurements per transaction. If more custom performance measurements are included in a transaction event, the list will be truncated.
```
Comment (vivianyentran) [inline docs/product/performance/metrics.mdx:123]:
```suggestion
You can define up to 10 custom performance measurements per transaction. If more custom performance measurements are included in a transaction event, the list will be truncated.
```
Not sure if we want to add a little more detail here about how the list is truncated? Do we keep the first 10 and drop the rest or something else? If a measurement is set on a span but gets truncated from the transaction, is it still queryable in discover/dashboard?

**Review by vivianyentran — COMMENTED** [review]
Had a couple questions on how its tied to transactions

File: docs/product/performance/metrics.mdx [hunk 4]
```diff
@@ -99,28 +99,25 @@ For each project, you can configure how [Apdex](#apdex) and [User Misery](#user-
...
 
-<Include name="custom-metrics-supported-sdks.mdx" />
+<Include name="custom-measurements-supported-sdks.mdx" />
 
-Custom metrics are accessible to you in [sentry.io](https://sentry.io) when you:
+Custom measurements are accessible to you in [sentry.io](https://sentry.io) when you:
 
 - Create visualizations in **Dashboards**
 - Query and add columns for them in **Discover**
-- View a transaction event detail (see image below)
```
Comment (vivianyentran) [inline docs/product/performance/metrics.mdx:120]: Are they viewable anywhere else right now after removing the event details view?

File: includes/custom-measurements-disclaimer.mdx (hunk 2 above)
Comment (vivianyentran) [reply includes/custom-measurements-disclaimer.mdx:3]: We should rewrite this since metrics will eventually be tied to spans

**Review by vivianyentran — APPROVED** [review]
Can we go ahead and push this change or are we waiting to time it with the launch?

File: docs/platforms/android/performance/instrumentation/performance-metrics.mdx [hunk 5]
```diff
@@ -8,29 +8,27 @@ Sentry's SDKs support sending performance metrics data to Sentry. These are nume
...
 <PlatformContent includePath="performance/automatic-performance-metrics" />
 
-## Custom Metrics
+## Custom Measurements
 
-In addition to automatic performance metrics, the SDK supports setting [custom performance metrics](/product/performance/metrics/#custom-performance-metrics) on transactions. This allows you to define metrics that are important to your application and send them to Sentry.
+<Include name="custom-measurements-disclaimer.mdx" />
 
-To set a performance metric, you need to supply the following:
+In addition to automatic performance metrics, the SDK supports setting [custom performance measurements](/product/performance/metrics/#custom-performance-measurements) on transactions. This allows you to define measurements that are important to your application and send them to Sentry.
```
[the remark this answers is not in the document]
Comment (matejminar) [reply docs/platforms/android/performance/instrumentation/performance-metrics.mdx:15]: Talked to JS folks, I believe custom measurements are tied to transactions. API was even changed to Sentry.setMeasurement.

File: docs/platforms/android/performance/instrumentation/performance-metrics.mdx [hunk 6]
```diff
@@ -8,29 +8,27 @@ Sentry's SDKs support sending performance metrics data to Sentry. These are nume
...
+
+To set a performance measurement, you need to supply the following:
 
 - name (`string`)
 - value (any numeric type - `float`, `integer`, etc.)
 - unit (`string`, Defaults to the string `none` if omitted.)
```
[the remark this answers is not in the document]
Comment (matejminar) [reply docs/platforms/android/performance/instrumentation/performance-metrics.mdx:21]: thanks! changed in all of the other files as well

File: includes/custom-measurements-disclaimer.mdx (hunk 2 above)
Comment (matejminar) [reply includes/custom-measurements-disclaimer.mdx]: applied 👍

File: docs/product/performance/metrics.mdx (hunk 3 above)
Comment (matejminar) [reply docs/product/performance/metrics.mdx]: If it's ok, I would leave that for other folks to add.

**Pull request merged**